README.md

```diff
@@ -1 +1,51 @@
-a lib to parse time specifications
+# dur
+
+a library to parse:
+* time specifications and convert them to durations.
+* datetime specifications and convert them to unix timestamps.
+
+Meant to work like [Graphite's from/until parsing](http://graphite.readthedocs.io/en/latest/render_api.html#from-until).
+which aims to support all `at(1)` formats also. So do we. see some notes below.
+
+## Durations
+
+Similar to [time.ParseDuration](https://golang.org/pkg/time/#ParseDuration) which has valid time units "ns", "us" (or "µs"), "ms", "s", "m", "h".
+which allows signed sequences and optional fractions like "300ms", "-1.5h" or "2h45m".
+
+But we work with larger units (and more forgiving ways to specify them) in this range:
+s/sec/secs/second/seconds, m/min/mins/minute/minutes, h/hour/hours, d/day/days, w/week/weeks, mon/month/months, y/year/years
+We however do not allow fractions or negative numbers.
+Note that this library takes a shortcut to provide convenience: a day = 24 hours, week = 7days, mon = 30 times 24 hours, year is 365 days.
+If these inaccuracies are a problem to you, do your own date math.
+
+E.g.: '7d', '1y6mon', '1hour15min3s'
+
+
+## Datetimes
+
+Functions to convert human specified date time specifications into timestamps.
+Supported formats:
+* `now-<duration>` now with duration subtracted. (for duration formats see above)
+* `-<duration>`: shortcut for the above
+* `now+<duration>` now with duration added. (for duration formats see above)
+* `HH:MM_YYMMDD`
+* `YYYYMMDD`
+* `MM/DD/YY` 
+* `integer`: unix timestamp
+* `<monthname>+<num>` like `january+3` or `march+8`
```

Comment:
`<monthname> <dayofmonth>` using `+` here is ambiguous since `+<duration>` is supported on any valid date specifier.